src/server/app.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask
+import logging
+#from flask_restful import Api, Resource, reqparse
+from flask_cors import CORS
+#from FlaskApp.api_layer.HelloApiHandler import HelloApiHandler
+from flask import abort, render_template, request, redirect, url_for, jsonify, json
+from DRL_parameters import DRL_parameters
+from GA_parameters import GA_parameters
+from Utils.utils import choose_data, read_data, sending_to_solver
+import numpy as np
+
+app = Flask(__name__)
+#app = Flask(__name__, static_url_path='', static_folder='react_client/build')
+CORS(app) #comment this on deployment
+#api = Api(app)
+
+@app.route('/test', methods = ['POST'] )
+def hello():
+    return request.form['hyper_params']
```

Comment:
Change this line to fix the server-side cross-site scripting vulnerability #9 
```suggestion
    return make_response(escape(request.form['hyper_params']))
```